src/trans/internal/abort_trans_mod.F90

```diff
@@ -13,8 +13,7 @@ MODULE ABORT_TRANS_MOD
 SUBROUTINE ABORT_TRANS(CDTEXT)
 
 USE TPM_GEN  , ONLY : NOUT,NERR
-USE TPM_DISTR, ONLY : NPROC,MYPROC
```

Comment:
This makes me wonder if we could remove all instances of `NPROC`, `MYPROC` from ecTrans.

Reply:
Yes I am not entirely sure. We can ask Mats. For an abort it is better with MPL. This was necessary to allow it to be part of ectrans_common so that other routines in this lib could use it.